certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+            "schema": {
+              "type": "object",
+              "properties": {
+                "boardid": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "boardid",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetBoards",
+                    "value-path": "board_id",
+                    "value-collection": "data",
+                    "value-title": "name"
+                  }
+                },
+                "taskid": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "taskid"
+                },
+                "title": {
+                  "type": "string",
+                  "description": "title"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description"
+                },
+                "priority": {
+                  "type": "string",
+                  "description": "priority",
+                  "title": "",
+                  "enum": [
+                    "Low",
+                    "Average",
+                    "High",
+                    "Critical"
+                  ]
+                },
+                "assignee": {
+                  "type": "string",
+                  "description": "assignee"
+                },
+                "color": {
+                  "type": "string",
+                  "description": "color"
+                },
+                "size": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "size"
+                },
+                "tags": {
+                  "type": "string",
+                  "description": "tags"
+                },
+                "deadline": {
+                  "type": "string",
+                  "description": "deadline"
+                },
+                "extlink": {
+                  "type": "string",
+                  "description": "extlink"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "type",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetTypes",
+                    "value-path": "name",
+                    "value-collection": "data",
+                    "value-title": "name"
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+	{
+		"propertyName": "Website",
+        "propertyValue": "https://kanbanize.com"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://kanbanize.com/privacy-policy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Kanban;Project Management;Agile"
```

Comment:
These categories are unfortunately invalid values, and are beyond the maximum 2 that we allow per connector. The valid values are:

AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website